ServerGUI.py

```diff
@@ -65,9 +68,30 @@ def __init__(self):
         self.drawPath()
         self.pid = PID(self.path, self.pathIndex, self.curr_tile.x, self.curr_tile.y)
         self.drawWayPoint(self.path[self.pathIndex])
+        if self.desired_heading is None:
+            self.updateDesiredHeading(self.path[self.pathIndex])
         self.main_loop()
         self.master.mainloop()
 
+    def processEndPoint(self, endPoint):
+        """
+        Processes the endpoint and returns the corresponding tuple.
+        Input examples:
+            ("forward", m) where m is in meters
+            ("turn", deg) where deg is positive for clockwise turns, negative for counterclockwise
+            (x, y) where x and y are coordinates to move to 
+        Updates self.endPoint to be the final coordinates to move to,
+        and updates self.desired_heading
+        """
+        if endPoint[0] == "forward":
+            self.endPoint = (self.curr_tile.x, self.curr_tile.y + endPoint[1])
```

Comment:
Do the units work out here? Idk if chatbots input is feet or meters. I know that we have out units set in consts.py